spk/sickchill/src/requirements.txt

```
@@ -1,88 +1,78 @@
-appdirs==1.4.4
-attrdict==2.0.1
-babelfish==0.5.5
-beautifulsoup4==4.9.3
-###beekeeper==0.9.2
+###py38 appdirs==1.4.4 
+babelfish==0.6.0
+beautifulsoup4==4.10.0
+beekeeper-alt==2021.7.16
 bencode.py==4.0.0
-CacheControl==0.12.6
-certifi==2020.11.8
-###cffi==1.14.3
-chardet==3.0.4
-click==7.1.2
-cloudscraper==1.2.48
+cachecontrol==0.12.6
+certifi==2021.5.30
```

```suggestion
certifi==2021.10.8
```

Let's make sure we have the latest version to make sure that Lets Encrypt certificateswork: https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

---

FYI The change only adds 2 CAs: https://github.com/certifi/python-certifi/commit/8effc0d8fd3e14e3db1f04c915246b82bb8c6107

1. Government of Tunisia, Agence National de Certification Electronique / National Digital Certification Agency (ANCE/NDCA) https://bugzilla.mozilla.org/show_bug.cgi?id=1587779
2. [harica](https://www.harica.gr/en/About/About) - https://bugzilla.mozilla.org/show_bug.cgi?id=1695486

---

Yes and yes on requirements changes. You guys are more in the know of these items than I am.

---

Na I just recently spend a lot of time diagnosing certificate failures from Lets encrypt (due to a root certificate expiring) and figured out common places where the trust store is embedded rather than using the one provided by the OS. And in Synology case we can argue that they were behind updating their store anyway. So turns out both methods are prone to failures due to infrequent updates.